website/docs/r/database_secrets_mount.md

```diff
@@ -0,0 +1,449 @@
+---
+layout: "vault"
+page_title: "Vault: vault_database_secrets_mount resource"
+sidebar_current: "docs-vault-resource-database-secrets-mount"
+description: |-
+  Configures any number of database secrets engines under a single mount resource
+---
+
+# vault\_database\_secrets\_mount
+
+Configure any number of database secrets engines under a single dedicated mount resource.
+
+~> **Important** All data provided in the resource configuration will be
```

Comment:
nit: I've only seen this paragraph on Datasource docs and haven't seen it on Resource docs

Reply:
interesting, I based the docs of off `database_secret_backend_connection.md`. We should probably revisit at a later date.